### PR TITLE
add new articles for using binaries

### PR DESCRIPTION
**Pull request by xu-yuan1**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

**Comment by xu-yuan1** [comment]
This PR is mainly focusing on the new content to be added. Removing the obsolete info is mainly in this one: https://github.com/openvinotoolkit/openvino/pull/12190

File: docs/install_guides/installing-openvino-from-archive-linux.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+   l_openvino_toolkit_<operating system>_dev_<release version>_<package ID>.tgz
+   l_openvino_toolkit_<operating system>_dev_<release version>_<package ID>.tgz.sha256
+   ``` 
+2. Open a command prompt terminal window. You can use the keyboard shortcut: Ctrl+Alt+T
+3. Change the directory to where you downloaded the archive files.<br>
+   For example, if you downloaded the files to the current user's `Downloads` directory, use the following command:
+   ```sh
+   cd ~/Downloads/
+   ```
+4. Verify the checksum of the downloaded `sha256` file:
```
Comment (helena-intel) [inline docs/install_guides/installing-openvino-from-archive-linux.md:69]: IMO this step (and downloading the sha256 file) should be optional. It is to confirm that the downloaded file is not corrupt. That is useful - but not essential. We do not currently do this for the installer either.

File: docs/install_guides/installing-openvino-from-archive-linux.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+   ```sh
+   sha256sum <archive name>.tgz.sha256
+   ```
+5. Extract the archive file:
+   ```sh
+   tar xf <archive name>.tgz -C <INSTALL_DIR>
+   ```
+   <br>You should see the installation has started.
```
Comment (helena-intel) [inline docs/install_guides/installing-openvino-from-archive-linux.md:77]: This is different with archives. After extracting, you have a directory <archive name>. OpenVINO is extracted to that directory. That is now your <INSTALL_DIR>

File: docs/install_guides/installing-openvino-from-archive-linux.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+   ```sh
+   sha256sum <archive name>.tgz.sha256
+   ```
+5. Extract the archive file:
+   ```sh
+   tar xf <archive name>.tgz -C <INSTALL_DIR>
+   ```
+   <br>You should see the installation has started. 
+
+By default, OpenVINO Runtime is installed to the following directory, referred to as `<INSTALL_DIR>` elsewhere in the documentation:
```
Comment (helena-intel) [inline docs/install_guides/installing-openvino-from-archive-linux.md:79]: For archives there is no default install directory. It is just whereever you unpacked it. That may be ~/Downloads, which is not ideal. We could recommend to move the directory to the previous standard places: 

For root or administrator: `sudo mv <archive> /opt/intel` and for regular users: `mv <archive> $HOME/intel`

File: docs/install_guides/installing-openvino-from-archive-linux.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+* For root or administrator: `/opt/intel/openvino_<version>/`
+* For regular users: `/home/<USER>/intel/openvino_<version>/`
+
+For simplicity, a symbolic link to the latest installation is also created: `/opt/intel/openvino_2022/` or `/home/<USER>/intel/openvino_2022/`.
```
Comment (helena-intel) [inline docs/install_guides/installing-openvino-from-archive-linux.md:83]: This does not happen automatically with archives. Users could do this themselves, it is useful. Something like: For simplicity, it is useful to create a symbolink link: `ln -s $HOME/intel/<archive> $HOME/intel/openvino_2022`. If a link already exists, remove the previous link with `rm $HOME/intel/openvino_2022`

File: docs/install_guides/installing-openvino-from-archive-linux.md (hunk 1 above)
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-from-archive-linux.md]: Thanks! I've made this step as optional.

File: docs/install_guides/installing-openvino-from-archive-linux.md (hunk 2 above)
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-from-archive-linux.md]: Updates. Would you check again? :)

File: docs/install_guides/installing-openvino-from-archive-linux.md (hunk 3 above)
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-from-archive-linux.md]: Implemented.

File: docs/install_guides/installing-openvino-from-archive-linux.md (hunk 4 above)
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-from-archive-linux.md]: Implemented. Do we have the commands for creating them on Windows and masOS too?

**Comment by xu-yuan1** [comment]
Hi @helena-intel , I've updated a lot according to the inputs from Artyom and Karol. Could you help check again to approve it?

**Review by kblaszczak-intel — APPROVED** [review]

File: docs/install_guides/installing-openvino-from-archive-windows.md [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+
+### <a name="model-optimizer">Step 3 (Optional): Download Additional Components</a>
+
+> **NOTE**: Since the OpenVINO™ 2022.1 release, the following development tools: Model Optimizer, Post-Training Optimization Tool, Model Downloader and other Open Model Zoo tools, Accuracy Checker, and Annotation Converter are not part of the installer. The OpenVINO™ Development Tools can only be installed via PyPI now. See [Install OpenVINO™ Development Tools](installing-model-dev-tools.md) for detailed steps.
+
+@sphinxdirective
+
+.. dropdown:: OpenCV
+
+   OpenCV is necessary to run demos from Open Model Zoo (OMZ). Some OpenVINO samples can also extend their capabilities when compiled with OpenCV as a dependency. The Intel® Distribution of OpenVINO™ provides a script to install OpenCV: ``<INSTALL_DIR>\extras\scripts\download_opencv.ps1``.
```
Comment (helena-intel) [inline docs/install_guides/installing-openvino-from-archive-windows.md:108]: This may change for 2022.2.

File: docs/install_guides/installing-openvino-from-archive-windows.md [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+@endsphinxdirective
+
+## <a name="get-started"></a>What's Next?
+
+Now you are ready to try out the toolkit.
+
+Developing in Python:
+   * [Start with tensorflow models with OpenVINO™](https://docs.openvino.ai/latest/notebooks/101-tensorflow-to-openvino-with-output.html)
+   * [Start with ONNX and PyTorch models with OpenVINO™](https://docs.openvino.ai/latest/notebooks/102-pytorch-onnx-to-openvino-with-output.html)
+   * [Start with PaddlePaddle models with OpenVINO™](https://docs.openvino.ai/latest/notebooks/103-paddle-onnx-to-openvino-classification-with-output.html)
```
Comment (helena-intel) [inline docs/install_guides/installing-openvino-from-archive-windows.md:165]: These notebooks will not work out of the box after following the steps in this guide. They require installing additional components and notebooks install instructions includes installation of openvino-dev (which includes openvino runtime).

File: docs/install_guides/uninstalling-openvino.md [hunk 7]
```diff
@@ -4,112 +4,28 @@
...
-  Use initial bootstrapper file ``m_openvino_toolkit_p_<version>.dmg`` to select product for uninstallation. Mount the file and double-click ``bootstrapper.app``. Follow the wizard instructions. Select **Remove** option when presented. If you have more product versions installed, you can select one from a drop-down menu in GUI.
-
-  .. image:: _static/images/openvino-uninstall-dropdown-macos.png
-    :width: 500px
-    :align: center
-
-@endsphinxdirective
-
-## Uninstall Using the Intel® Software Installer
+If you have installed OpenVINO Runtime from archive files, use the following commands to uninstall it:
```
Comment (helena-intel) [inline docs/install_guides/uninstalling-openvino.md:7]:
```suggestion
If you have installed OpenVINO Runtime from archive files, you can uninstall it by deleting the zip file and extracted folder.
```

File: docs/install_guides/uninstalling-openvino.md [hunk 8]
```diff
@@ -4,112 +4,28 @@
...
-
-  Alternatively, follow the steps:
-  
-  1. Go to installation directory.
-  2. In ``OpenVINO`` directory find ``Installer`` folder and open it.
-  3. Double-click on ``installer.exe`` and you will be presented with dialog box shown below.
+  You can choose either of the following methods:
 
-.. tab:: Linux
+  * Use the Windows GUI to remove the extracted files.
```
Comment (helena-intel) [inline docs/install_guides/uninstalling-openvino.md:14]:
```suggestion
  * Use Windows Explorer to remove the files.
```

File: docs/install_guides/uninstalling-openvino.md [hunk 9]
```diff
@@ -4,112 +4,28 @@
...
+  * Open the Command window and run:
```
Comment (helena-intel) [inline docs/install_guides/uninstalling-openvino.md:15]:
```suggestion
  * Open a Command Prompt and run:
```

File: docs/install_guides/uninstalling-openvino.md [hunk 10]
```diff
@@ -4,112 +4,28 @@
...
 
-  1. Run the installer file from the user mode installation directory:
-   
-  .. code-block:: sh
+     .. code-block:: sh
   
-    /home/<user>/intel/openvino_installer/installer
-
-  or in a case of administrative installation:
+       rmdir <extracted_folder>
```
Comment (helena-intel) [inline docs/install_guides/uninstalling-openvino.md:19]:
```suggestion
       rmdir /s <extracted_folder>
```
Without `/s` you will get the error that the directory is not empty

**Review by helena-intel — CHANGES_REQUESTED** [review]

File: docs/install_guides/installing-openvino-from-archive-windows.md [hunk 11]
```diff
@@ -0,0 +1,196 @@
+# Install OpenVINO™ Runtime on Windows 10 from an Archive File {#openvino_docs_install_guides_installing_openvino_from_archive_windows}
+
+With the OpenVINO™ 2022.2 release, you can download the archive files and install OpenVINO Runtime directly.
```
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-from-archive-windows.md:3]: Here we can take "install" as the purpose and result of the unzipping process. It does not have to be a specific action.

[the remark this answers is not in the document]
Comment (helena-intel) [reply docs/install_guides/installing-openvino-from-archive-windows.md:3]: For me the current phrasing is somewhat confusing. Referring to archives as an installation method is fine; it is mostly the "install directly" part that makes me wonder if the other methods install indirectly and what the difference is.

File: docs/install_guides/installing-openvino-from-archive-windows.md [hunk 12]
```diff
@@ -0,0 +1,196 @@
...
+
+  Processor graphics are not included in all processors. 
+  See `Product Specifications`_ for information about your processor.
+  
+  .. _Product Specifications: https://ark.intel.com/
+
+.. tab:: Software
+
+  * `Microsoft Visual Studio 2019 with MSBuild <http://visualstudio.microsoft.com/downloads/>`_
+  * `CMake 3.14 or higher, 64-bit <https://cmake.org/download/>`_
```
[the remark this answers is not in the document]
Comment (helena-intel) [reply docs/install_guides/installing-openvino-from-archive-windows.md:37]: @ilya-lavrenov  is CMake a requirement for C++ developers? If it is only for building samples, we should specify that.

File: docs/install_guides/installing-openvino-from-archive-windows.md [hunk 13]
```diff
@@ -0,0 +1,196 @@
...
+
+### <a name="install-openvino"></a>Step 1: Download and Install OpenVINO Core Components
+
+1. Select and download the OpenVINO™ archive files from [Intel® Distribution of OpenVINO™ toolkit download page](https://software.intel.com/en-us/openvino-toolkit/choose-download).
+   There are typically two files for you to download: 
+   ```sh
+   w_openvino_toolkit_<operating system>_dev_<release version>_<package ID>.zip
+   w_openvino_toolkit_<operating system>_dev_<release version>_<package ID>.zip.sha256
+   ``` 
+   where the `.sha256` file is used to verify the success of the download process.
```
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-from-archive-windows.md:62]: It's not optional for Linux either. It depends on whether we want users to check the download. If we want them to do that, we can list this step although it's not always required.

**Review by artanokhov — APPROVED** [review]

File: docs/install_guides/installing-openvino-from-archive-windows.md (hunk 5 above)
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-from-archive-windows.md:112]: For now, I think we can redirect users to the Github page. Updated.

File: docs/install_guides/installing-openvino-from-archive-windows.md (hunk 6 above)
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-from-archive-windows.md:171]: hmmm, maybe we should recommend the ones that can work in a browser with binder. What do you think?

File: docs/install_guides/pypi-openvino-rt.md [hunk 14]
```diff
@@ -78,9 +78,9 @@ If installation was successful, you will not see any error messages (no console
 
 For general troubleshooting steps and issues, see [Troubleshooting Guide for OpenVINO Installation](https://docs.openvino.ai/2022.2/openvino_docs_get_started_guide_troubleshooting.html). The following sections also provide explanations to several error messages. 
 
-### Error: Microsoft Visual C++ 14.0 is required. Get it with "Build Tools for Visual Studio"
+### ERROR:root:Could not find the Inference Engine or nGraph Python API.
 
-On Windows* some dependencies may require compilation from source when installing. To resolve this issue, you need to install [Build Tools for Visual Studio* 2019](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019) and repeat package installation.
+On Windows*, some libraries are necessary to run OpenVINO. To resolve this issue, install the [C++ redistributable (.exe)](https://aka.ms/vs/17/release/vc_redist.x64.exe). You can also view a full download list on the [official support page](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist).
```
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/pypi-openvino-rt.md:83]: This is in the troubleshooting section so it's only one of the errors that can occur. For macOS/Linux, we need more tests to get the errors and solutions, which is not that necessary. I will add C++ redistributable to the System Requirements section.

File: docs/install_guides/installing-openvino-from-archive-windows.md (hunk 6 above)
Comment (helena-intel) [reply docs/install_guides/installing-openvino-from-archive-windows.md:171]: Binder is great to quickly get started, but performance and availability are limited so it would be good to also refer to the installation instructions for the notebooks as an optional step. Also because after installing you may want to know how to get things working locally (because you do not want to develop your own python scripts in Binder). But it is a bit unfortunate that at the moment notebooks installation is a separate process that requires many additional dependencies.  I agree we should not have that here as a mandatory step when you think you're done with installing. 

In the future we could have a way to get started with a subset of notebooks that does not require additional installation. The hello world notebooks (000 series) should work out of the box after this installation if someone has Jupyter installed.

File: docs/install_guides/installing-openvino-from-archive-windows.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (helena-intel) [reply docs/install_guides/installing-openvino-from-archive-windows.md:65]: IMO it should be optional everywhere. We do not currently have this step for our installation methods though all these files could technically be corrupt.

File: docs/install_guides/configurations-for-intel-gpu.md [hunk 15]
```diff
@@ -12,7 +12,7 @@ This page introduces additional configurations for Intel® Processor Graphics (G
 
 ## Linux
 
-If you have installed OpenVINO Runtime via the installer, APT, or YUM, follow these steps to work with GPU:
+If you have installed OpenVINO Runtime from the archive file, follow these steps to work with GPU:
```
Comment (helena-intel) [inline docs/install_guides/configurations-for-intel-gpu.md:15]: image
I do not have this install_dependencies folder in the prerelease Windows archive for 2022.1.1 @artanokhov

**Review by helena-intel — APPROVED** [review]
Thanks @xu-yuan1 ! I have a few questions left, but am approving the PR now so you are not blocked from merging while I am away. I hope @zKulesza can UAT test the new instructions.

File: docs/install_guides/installing-openvino-from-archive-windows.md [hunk 16]
```diff
@@ -0,0 +1,196 @@
...
+  Processor graphics are not included in all processors. 
+  See `Product Specifications`_ for information about your processor.
+  
+  .. _Product Specifications: https://ark.intel.com/
+
+.. tab:: Software
+
+  * `Microsoft Visual Studio 2019 with MSBuild <http://visualstudio.microsoft.com/downloads/>`_
+  * `CMake 3.14 or higher, 64-bit <https://cmake.org/download/>`_
+  * `Python 3.6 - 3.9, 64-bit <https://www.python.org/downloads/windows/>`_
```
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-from-archive-windows.md]: Updated

File: docs/install_guides/installing-openvino-from-archive-windows.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-from-archive-windows.md]: Got your point. Updated the wording. :)

File: docs/install_guides/installing-openvino-from-archive-windows.md [hunk 17]
```diff
@@ -0,0 +1,196 @@
...
+   * If the values are the same, you have downloaded the correct file successfully.
+   * If not, create a Support ticket [here](https://www.intel.com/content/www/us/en/support/contact-intel.html).
+
+4. Unzip the `<archive name>.zip` file using your preferred archive tool. 
+ 
+The standard OpenVINO `<INSTALL_DIR>` used in this document is `C:\Program Files (x86)\Intel\openvino_<version>\`. You're recommended to move the extracted files to that directory. 
+
+For simplicity, you can also create a symbolic link to the latest installation: `C:\Program Files (x86)\Intel\openvino_2022\`.
+
+To check **Release Notes** please visit: [Release Notes](https://software.intel.com/en-us/articles/OpenVINO-RelNotes).
```
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-from-archive-windows.md]: Updated.

**Comment by helena-intel** [comment]
This PR is for the 2022.2 release branch and contains big changes that are very specific to that release. It should not be merged to master before the 2022.2 release.

**Comment by xu-yuan1** [comment]
> This PR is for the 2022.2 release branch and contains big changes that are very specific to that release. It should not be merged to master before the 2022.2 release.

Yes this is 2022.2 only. Master keeps the 2022.1 version.

File: docs/install_guides/installing-openvino-from-archive-windows.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-from-archive-windows.md]: According to @ilya-lavrenov , CMake is only required for building sample applications. Updated.

File: docs/install_guides/installing-openvino-from-archive-windows.md (hunk 6 above)
Comment (xu-yuan1) [reply docs/install_guides/installing-openvino-from-archive-windows.md]: Updated.
